tests/sentry/api/endpoints/test_organization_member_index.py

```diff
@@ -498,7 +498,41 @@ def test_cannot_invite_retired_role_with_flag(self):
 class OrganizationMemberPermissionRoleTest(OrganizationMemberListTestBase, HybridCloudTestMixin):
     method = "post"
 
+    def test_owner_invites(self):
+        self.organization.flags.disable_member_invite = True
+        self.organization.save()
+
+        owner_user = self.create_user("owner@localhost")
+        self.owner = self.create_member(
+            user=owner_user, organization=self.organization, role="owner"
+        )
+        self.login_as(user=owner_user)
+
+        data = {"email": "eric1@localhost", "role": "owner", "teams": [self.team.slug]}
```

Comment:
nit: We can move all these constants into a helper that generates a random email and takes in the role.

If you wanna be fancy I think you could somehow move these all into a loop and use OrganizationMember.get_allowed_org_roles_to_invite() to know the error code and pass it into `self.get_response`